Automattic/woocommerce-payments
# Update the minimum WC version used in e2e and phpunit in GH workflows

#### Changes proposed in this Pull Request
Update the minimum WC version in GH flows from 6.2.2 to 6.4.1.

This includes the same changes from [this other PR](https://github.com/Automattic/woocommerce-payments/pull/4650), but now pointing to develop.

Updating the version in the oldest compatibility PHP unit setup is pending. One test is failing when the minimum version there is updated. To be addressed separately.

#### Testing instructions

* Confirm that the GH flows for PHP unit tests pass with no errors
* Confirm that 6.2.2 is replaced everywhere in the workflows by 6.4.1 (except for [this instance](https://github.com/Automattic/woocommerce-payments/blob/develop/.github/workflows/compatibility.yml#L21) which is going to be addressed separately)

-------------------


**Post merge**
